### PR TITLE
Fix some typos.

### PR DESCRIPTION
**Pull request by waywardmonkeys**

### CODE REVIEW

File: tools/system_libs.py [hunk 1]
```diff
@@ -963,7 +963,7 @@ class libc(MuslInternalLibrary,
 
   # Disable certain warnings for code patterns that are contained in upstream musl
   cflags += ['-Wno-ignored-attributes',
-             # tre.h defines NDEBUG internally itself
+             # tree.h defines NDEBUG internally itself
```
Comment (kripken) [inline tools/system_libs.py:966]: I think this was correct before actually, it just looks odd :smile: 

https://github.com/emscripten-core/emscripten/blob/288d475aa23ea1f7068b24adc719ff4b12ed2f51/system/lib/libc/musl/src/regex/tre.h#L38

Maybe TRE is an acronym..?

**Review by kripken — COMMENTED** [review]
Thanks! lgtm % comment

**Review by kripken — APPROVED** [review]

File: tools/system_libs.py (hunk 1 above)
Comment (waywardmonkeys) [reply tools/system_libs.py]: Fixed and force pushed. Thanks for the catch!

**Comment by sbc100** [comment]
I see you don't like my unique way of spelling things!

**Pull request merged**